tests/tracer/telemetry/test_request.py

```diff
@@ -0,0 +1,91 @@
+import mock
+
+from ddtrace.internal.telemetry.data import get_application
+from ddtrace.internal.telemetry.data import get_host
+from ddtrace.internal.telemetry.request import create_telemetry_request
+from ddtrace.settings import _config as config
+
+
+def test_create_telemetry_request():
+    """validates the return value of _create_telemetry_request"""
+    with mock.patch("ddtrace.internal.telemetry.request.monotonic") as t:
+        t.return_value = 888366600
+        with mock.patch("ddtrace.internal.telemetry.request.get_runtime_id") as get_rt_id:
+            get_rt_id.return_value = "1234-567"
```

Comment:
I removed the other `create_telemetry_request` tests since they did not add any additional value (each test just passed in 3 arguments and asserted the same fields were set in the dictionary